front/client/components/Article.vue

```diff
@@ -74,17 +104,35 @@ export default class ArticleItem extends Vue {
     // return this.$store.state.categoryItem.categoryItemList
     return this.$store.state.categoryItem
   }
-
+// TODO - Andreys changes - implement! try 3d option or completely rewrite - still bug exists
   get getCategoryDescription() {
-    const description = this.$store.state.categoryItem.categoryItemList.description.html;
-    return description
+    if (!this.$store.state.categoryItem.categoryItemList.description === undefined) {
```

Comment:
can be written as if (!this.$store.state.categoryItem.categoryItemList.description) {